### PR TITLE
[StatsCoordinator] Dont use string interpolation when constructing qu…

### PR DESCRIPTION
**Pull request by segiddins**
…tries

\c @floere @orta

### CODE REVIEW

**Comment by orta** [comment]
I don't understand what's going on, so I'm happy to leave it to @floere

**Comment by floere** [comment]
I would not want to take away such a marvellous learning experience from you! Back atcha, @orta :)

File: runner/stats_coordinator.rb [hunk 1]
```diff
@@ -96,13 +96,13 @@ def target pod_name, type, time=nil
...
-        AND product_type = '#{type_id}'
+        WHERE dependency_name = $1
+        AND product_type = $2
         AND pod_try = false
       eos
-      query += "AND sent_at >= current_date - interval '#{time}'" if time
+      query << "AND sent_at >= current_date - interval $3" if time
 
-      @connection.exec(query)[0]["count"].to_i || 0
+      @connection.exec(query, [pod_name, type_id, time])[0]["count"].to_i || 0
```
Comment (kylef) [inline runner/stats_coordinator.rb]: :+1: * :100:

**Comment by segiddins** [comment]
This prevents the possibility of SQL injection

**Pull request merged**